docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/type/string-tensor-unpack-15.rst

```diff
@@ -0,0 +1,154 @@
+.. {#openvino_docs_ops_type_StringTensorUnpack_15}
+
+StringTensorUnpack
+===================
+
+
+.. meta::
+  :description: Learn about StringTensorUnpack-15 - operation which unpacks a batch of strings into three tensors.
+
+**Versioned name**: *StringTensorUnpack-15*
+
+**Category**: *Type*
+
+**Short description**: *StringTensorUnpack* operation transforms a given batch of strings into three tensors - one containing 
+the concatenated string data, and two other storing begin and end indices of the strings, respectively.
+
+**Detailed description**
+
+Consider an ``input`` string tensor containing values ``["Intel", "OpenVINO"]``.
+
+The operator will transform the tensor into three outputs:
+
+* *begins* = [0, 5]
+    * ``begins[0]`` is equal to 0, because the first string starts at the beggining index.
+    * ``begins[1]`` is equal to 5, because length of the string "Intel" is equal to 5.
+    * ``begins.shape`` is equal to [2], because the ``input`` is a batch of 2 strings.
+
+* *ends* = [5, 13]
+    * ``ends[0]`` is equal to 5, because length of the string "Intel" is equal to 5.
+    * ``ends[1]`` is equal to 13, because length of the string "OpenVINO" is 8, and it needs to be summed up
+    with length of the string "Intel".
+    * ``ends.shape`` is equal to ``[2]``, because the ``input`` is a batch of 2 strings.
```

Comment:
```suggestion
    * ``ends[1]`` is equal to 13, because length of the string "OpenVINO" is 8, and it needs to be summed up with length of the string "Intel".
    * ``ends.shape`` is equal to ``[2]``, because the ``input`` is a batch of 2 strings.
```
Last element in list is not properly formatted in generated docs